packages/expo-constants/src/Constants.ts

```diff
@@ -29,6 +29,22 @@ export {
   WebManifest,
 };
 
+/**
+ * Helper method that extracts any custom properties in the `extra` part of the Expo config.
+ * The `eas` property is excluded (reserved for Expo internal use).
+ * @param config The config from which to extract properties.
+ * @return Object containing any properties found. If no extra properties found, returns an empty object.
+ */
+export const customPropertiesFromExpoConfig = (config: ExpoConfig) => {
+  const result: { [key: string]: any } = {};
+  for (const key in config.extra) {
+    if (key !== 'eas') {
```

Comment:
I don't think we should exclude the `eas` key (it may be helpful for people to put it in a debug screen or something).

(if we agree on this point, then this method would simply be `return config.extra`, which to me a method seems like overkill for just a property access)